test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/BasicSearchTests.cs

```diff
@@ -924,6 +924,14 @@ public async Task GivenASearchRequestWithValidParametersAndStrictHandling_WhenHa
             Assert.Equal(HttpStatusCode.OK, response.StatusCode);
         }
 
+        [Fact]
+        public async Task GivenACompositeTokenNumberNumberSearchParameter_WhenSearching_ReturnsSearchResults()
```

Comment:
`NumberNumber`?

Reply:
It's a composite search parameters of 3 components. First is token and rest two is numbers. So `NumberNumber`